store/listings/management/commands/fetch_listings.py

```diff
@@ -37,6 +40,17 @@ def handle(self, *args, **options):
                 'filename': 'cotton_eur.txt'
             },
         ]
-
         for listings in listings_data:
-            save_to_file(listings=listings)
+            for listing in listings['data']:
+                listing_object = Listing.objects.create(
+                    title=listing.get('title'),
+                    description=listing.get('description'),
+                    price=listing.get('price'),
+                    currency=listing.get('currency'),
+                    category=listings['category']
+                )
+
+                listing_object.uid = listing.get('listing_id')
+                listing_object.save()
```

Comment:
Listing object creation code should be wrapped with `atomic` context or decorator (whatever you prefer)

`from django.db.transaction import atomic`

If there's an exception, db changes are rolled back.

Right now, if there's an exception, data is still saved and can cause inconsistencies in db.